client/py/examples/control/press/control_sequence.py

```diff
@@ -7,10 +7,23 @@
 #  License, use of this software will be governed by the Apache License, Version 2.0,
 #  included in the file licenses/APL.txt.
 
+"""
+This example simulates a basic control sequence where a tank with pressure represented
+by sensor_0 is pressurized in 20 psi increments. The tank is then vented after holding
```

Comment:
Make sure to compensate for the new channel names here (`press_pt`, `press_vlv_cmd`, and `vent_vlv_cmd`)